reader/src/main/java/org/kaazing/monitoring/reader/api/GatewayCounters.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Doesn't look as if this class is used anywhere. If it's dead code please remove it.